en/modules/openiddict.md

```diff
@@ -0,0 +1,462 @@
+# OpenIddict Module
+
+This module provides integration and management functionality for Identity Server;
+
+* Built on the [OpenIddict-core](https://github.com/openiddict/openiddict-core) library.
+* Manage **Application** and **API resources** in the system. 
+* Set **permissions** for clients.
+* Manage **API resources**
+
+See [the module description page](https://commercial.abp.io/modules/Volo.openiddict.Ui) for an overview of the module features.
+
+## How to Install
+
+OpenIddict is pre-installed in [the startup templates](../Startup-Templates/Index) from version 6.0.0-rc1. So, no need to manually install it.
+
+## Packages
+
+This module follows the [module development best practices guide](https://docs.abp.io/en/abp/latest/Best-Practices/Index) and consists of several NuGet and NPM packages. See the guide if you want to understand the packages and relations between them.
+
+You can visit [Identity module package list page](https://abp.io/packages?moduleName=Volo.Identity.Pro) to see list of packages related with this module.
```

Comment:
Text and link are wrong. This is not Identity module.

Reply:
I thought about the link between Identity module.
IdentityServer docs has the same info: 
https://github.com/abpio/abp-commercial-docs/blob/4ce1975088eefe3ebfc9d8d7efaf3d8b3e6c7b35/en/modules/identity-server.md?plain=1#L22